File: 프로그래머스/unrated/181880. 1로 만들기/1로 만들기.py
def solution(num_list):
    
    cnt = 0
    for num in num_list:
        while num != 1:
            cnt += 1
            if num % 2 == 0:
                num //= 2    
            else:
                num -= 1
                num //= 2
                
    return cnt

File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.java
class Solution {
    public int solution(int[] num_list) {
        
        int ans = 0;
        for(int i = 0; i < num_list.length; i++) {
            ans += calcCnt(num_list[i]);
        }
        
        return ans;
    }
    
    public static int calcCnt(int num) {
        
        int cnt = 0;
        while(num > 1) {
            cnt++;
            if (num % 2 == 0) {
                num /= 2;
            } else {
                num--;
                num /= 2;
            }
        }
        
        return cnt;
        
    }
}